src/pages/product/card/ProductCard.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useWishlist } from '../../wishlist/wishlist-context';
+import "../../../App.css";
+function ProductCard({
+    _id,
+    title,
+    author,
+    price,
+    inStock,
+    rating,
+    productImage,
+    isFastDelivery,
+    categoryName
+})
+ {
+
+    const {wishlist_dispatch} = useWishlist();
+    function wishListHandler(){
+      wishlist_dispatch({type:"ADD-TO-WISHLIST", 
```

Comment:
I would suggest to use constants in action type names as this would make it less volatile to spelling mistakes.

Reply:
Ok 👍🏻